lib/rules/syntaxes/slot-attribute.js

```diff
@@ -11,6 +11,12 @@ module.exports = {
   supported: '<3.0.0',
   /** @param {RuleContext} context @returns {TemplateListener} */
   createTemplateBodyVisitor(context) {
+    const options = context.options[0] || {}
+    /** @type {string[]} */
+    const ignore = Array.isArray(options.ignore)
+      ? options.ignore
+      : [options.ignore]
```

Comment:
Please make this a `Set`; it can be queried with `set.has()` in constant time (instead of `array.includes()` which takes linear time).

Reply:
Fixed: [847a098](https://github.com/vuejs/eslint-plugin-vue/pull/2314/commits/847a0986d974e78ed66f21ce03857365a261b329)